guides/cookbook.md

```diff
@@ -1450,13 +1889,61 @@ previous sub-section.
 Not currently supported.
 </div>
 
+<div class="tabpane Python" markdown="1">
+Instances of [SymbolTableCatalog](https://ion-python.readthedocs.io/en/latest/amazon.ion.html#amazon.ion.symbols.SymbolTableCatalog)
+are used by readers to resolve shared symbol table imports encountered in Ion data. Any
+shared symbol tables that are expected to be encountered in the stream should be registered
+with the `SymbolTableCatalog` that is provided to the reader.
+
+```python
+from amazon.ion.symbols import SymbolTableCatalog
+
+table = shared_symbol_table(u'test.csv.columns', 1, (u'id', u'type', u'state'))
+catalog = SymbolTableCatalog()
+catalog.register(table)
+```
+
+#### Using simpleion
+`load/loads` accept a catalog via the `catalog` parameter.
+
+```python
+data = ... # The Ion representation of the CSV data containing a shared sybol table import.
+values = simpleion.loads(data, catalog=catalog, single_value=False)
+assert values[2][u'id'] == 3
+```
+
+#### Using events
+The `managed_reader` coroutine accepts a catalog via the `catalog` parameter.
+
+```python
+data = BytesIO(...) # The Ion representation of the CSV data containing a shared sybol table import.
```

Comment:
imports?  can we provide actual data here?

Reply:
Done.